src/maintainer/adding_pkgs.rst

```diff
@@ -556,7 +556,7 @@ You can include files required for testing with the ``source_files`` section:
       imports:
         - package_name
       requires:
-        - pytest tests test_pkg_integration.py
```

Comment:
We should add this line back as a commands section.

Reply:
done, placed after source_files